src/OpenSSL.PrivateKeyDecoder/DecoderUtils.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.IO;
+
+namespace OpenSSL.PrivateKeyDecoder
+{
+    internal static class DecoderUtils
+    {
+        internal static bool CompareByteArrays(byte[] arrayA, byte[] arrayB)
+        {
```

Comment:
Replace content from this method by https://msdn.microsoft.com/en-us/library/bb348567.aspx if possible.

Reply:
We can't do this because the library supports .Net Framework 2.0 and Linq is not supported.